tests/kafkatest/sanity_checks/test_performance_services.py

```diff
@@ -31,14 +31,8 @@ def __init__(self, test_context):
         self.num_records = 10000
         self.topic = "topic"
 
-        self.zk = ZookeeperService(test_context, 1) if quorum.for_test(test_context) == quorum.zk else None
```

Comment:
Could you please remove the import `from kafkatest.services.zookeeper import ZookeeperService`?